### PR TITLE
Add autoincrement property to migration file (programming exercise test cases)

### PR DESCRIPTION
**Pull request by thilo-behnke**
### Checklist
- [x] I tested the changes and all related features on the test server https://artemistest.ase.in.tum.de.
- [x] ~I documented my source code using the JavaDoc / JSDoc style.~
- [x] ~I added integration test cases for the server (Spring) related to the features~
- [x] ~I added integration test cases for the client (Jest) related to the features~
- [x] ~I added screenshots/screencast of my UI changes~
- [x] ~I translated all the newly inserted strings~

### Description

When we rolled out the programming exercise test case commit to production, auto increment was not working for the test cases.
The issue was that the autoincrement property generated by jhipster was removed from the changelog file when I did the merge of the 2 changelog files.

:warning: Note: We need to clear the checksums on the test and production server when this is merged. :warning:

### CODE REVIEW

File: src/main/resources/config/liquibase/changelog/20190603142500_changelog.xml [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
-    <!-- add sequential test run boolean flag to exercise -->
-    <addColumn tableName="exercise">
-        <column name="sequential_test_runs" type="Boolean"/>
-    </addColumn>
-</changeSet>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <property name="autoIncrement" value="true"/>
```
Comment (thilo-behnke) [inline src/main/resources/config/liquibase/changelog/20190603142500_changelog.xml]: This is the only line I changed, but I also ran an intellij autoformat (prettier does not support XML formatting).

**Comment by thilo-behnke** [comment]
This PR is verified on the server, it worked after the clear checksum command was run.

File: src/main/resources/config/liquibase/changelog/20190603142500_changelog.xml (hunk 1 above)
Comment (krusche) [reply src/main/resources/config/liquibase/changelog/20190603142500_changelog.xml]: Can you revert the auto formatting please?

Comment (thilo-behnke) [reply src/main/resources/config/liquibase/changelog/20190603142500_changelog.xml]: I reverted the autoformat.

**Pull request merged**